main.ts

```diff
@@ -297,7 +306,7 @@ export const main = async (args: ParsedArgs) => {
       true
     )
 
-    process.exit()
+    process.exit(hadErrors ? -1 : 0)
```

Comment:
The error behaviour here isn't great. We should probably print them all again.
But the global error file makes that a bit inconvenient. Should be another PR to rework that into project based error handling.